Samples/ForeignAccelerometer/README.md

```diff
@@ -0,0 +1,5 @@
+# ForeignAccelerometer
```

Comment:
Did this branch add the ForeignAccelerometer exmaple, or was that on a branch or something?